collector.spec

```diff
@@ -0,0 +1,54 @@
+%global goipath         github.com/os-observability/redhat-opentelemetry-collector
+
+Version:                0.92.0
+ExcludeArch:            %{ix86} s390 ppc ppc64 aarch64
+
+%gometa
+
+%global common_description %{expand:
+TODO}
+
+%global golicenses    LICENSE
+%global godocs        README.md
+
+Name:           redhat-opentelemetry-collector
+Release:        1%{?dist}
+Summary:        TODO
+
+License:        Apache-2.0
+
+Source0:        %{name}-%{version}.tar.gz
+Source1:        %{name}-deps-%{version}.tar.gz
+
+BuildRequires: go-toolset-1.20
+BuildRequires: git
+
+%description
+%{common_description}
+
+%gopkg
+
+%prep
+%goprep
+
+%build
+make build
```

Comment:
Does `make` add some additonal `BuildRequires`?

Reply:
I guess so? https://github.com/rhinstaller/anaconda/blob/master/anaconda.spec.in#L61